docs/en/compute/Compute-Presto.md

```diff
@@ -27,8 +25,9 @@ latency especially when data is remote or network is slow or congested.
 * [Setup Java](Java-Setup.html) for Java 8 Update 60 or higher (8u60+), 64-bit.
```

Comment:
`[Setup Java](Java-Setup.html) for Java 8 Update 60 or higher (8u60+), 64-bit.` ->  `Setup Java for Java 8 Update 60 or higher (8u60+), 64-bit.`

Reply:
fixed